setup.py

```diff
@@ -14,60 +11,41 @@
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
-
-# Fortran extensions for extra speed (optional)
-ext_fstat = Extension(name="lifelines._utils._cindex",
-                      sources=["lifelines/_utils/_cindex.f90"])
-exts = [ext_fstat]
-
 exec(compile(open('lifelines/version.py').read(),
                   'lifelines/version.py', 'exec'))
 
-done = False
-# First try to build with the extension. In case of failure, do without.
-for ext_modules in [exts, []]:
-    if done:
-        break
-
-    try:
-        setup(
-            name="lifelines",
-            version=__version__,
-            author="Cameron Davidson-Pilon, Jonas Kalderstam",
-            author_email="cam.davidson.pilon@gmail.com",
-            description="Survival analysis in Python, including Kaplan Meier, Nelson Aalen and regression",
-            license="MIT",
-            keywords="survival analysis statistics data analysis",
-            url="https://github.com/CamDavidsonPilon/lifelines",
-            packages=['lifelines',
-                      'lifelines.datasets',
-                      'lifelines._utils',
-                      'lifelines.tests'],
-            long_description=read('README.txt'),
-            classifiers=[
-                "Development Status :: 4 - Beta",
-                "License :: OSI Approved :: MIT License",
-                "Programming Language :: Python",
-                "Topic :: Scientific/Engineering",
-                ],
-            install_requires=[
-                "numpy",
-                "scipy",
-                "pandas>=0.14",
-            ],
-            package_data={
-                "lifelines": [
-                    "*.f90",
-                    "../README.md",
-                    "../README.txt",
-                    "../LICENSE",
-                    "../MANIFEST.in",
-                    "../*.ipynb",
-                    "datasets/*",
-                ]
-            },
-            ext_modules=ext_modules
-        )
-        done = True
-    except:
-        print("Failed to build. Trying again without native extensions.")
+setup(
+    name="lifelines",
+    version=__version__,
+    author="Cameron Davidson-Pilon, Jonas Kalderstam",
+    author_email="cam.davidson.pilon@gmail.com",
+    description="Survival analysis in Python, including Kaplan Meier, Nelson Aalen and regression",
+    license="MIT",
+    keywords="survival analysis statistics data analysis",
+    url="https://github.com/CamDavidsonPilon/lifelines",
+    packages=['lifelines',
+              'lifelines.datasets',
+              'lifelines.tests'],
+    long_description=read('README.txt'),
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python",
+        "Topic :: Scientific/Engineering",
+        ],
+    requires=[
+        "numpy",
+        "scipy",
+        "pandas(>=0.14)",
```

Comment:
I read the [distutils documentation](https://docs.python.org/2/distutils/setupscript.html), which states:

> If specific versions are required, a sequence of qualifiers can be supplied in parentheses. Each qualifier may consist of a comparison operator and a version number.

When I tried to run it without the parens (as it was before), it gave me a `ValueError: expected parenthesized list: '>=0.14'`. I thought this was because I changed the `numpy.distutils.core` line to `distutils.core` (since we don't need Fortran stuff anymore), but I just tried changing it back and it still doesn't work for me. So I'm not sure what's going on.